src/Symfony/Component/Console/Output/AnsiColorCode.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Output;
+
+/**
+ * @author Julien Boudry <julien@condorcet.vote>
+ */
+enum AnsiColorCode
+{
+    /*
```

Comment:
phpdoc should start with `/**`

Reply:
Done. But Fabbot is in a bad mood with this. In fact, it was his previous patch that introduced this.  Maybe a bad support of Enums.

Reply:
Ignore fabbot on this. PHP-CS-Fixer might not have full support for enums yet